File: VB/DXApplication5/MVVM/Views/FindDialogView.vb
Imports Microsoft.VisualBasic
Imports DXApplication5.MVVM.ViewModels
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace DXApplication5.MVVM.Views
	Partial Public Class FindDialogView
		Inherits DevExpress.XtraEditors.XtraUserControl
		Public Sub New()
			InitializeComponent()
        End Sub
        Protected Overrides Sub OnLoad(e As EventArgs)
            MyBase.OnLoad(e)
            InitializeBindings()
        End Sub
		Private Sub InitializeBindings()
			Dim fluent = mvvmContext.OfType(Of FindDialogViewModel)()
            fluent.SetBinding(findTextEdit, Function(f) f.EditValue, Function(x) x.SearchID)
		End Sub
	End Class
End Namespace

File: CS/DXApplication5/MVVM/Views/FindDialogView.cs
using DXApplication5.MVVM.ViewModels;
using System;
using System.Collections.Generic;
using System.Linq;

namespace DXApplication5.MVVM.Views {
    public partial class FindDialogView : DevExpress.XtraEditors.XtraUserControl {
        public FindDialogView() {
            InitializeComponent();
        }
        protected override void OnLoad(EventArgs e) {
            base.OnLoad(e);
            InitializeBindings();
        }
        private void InitializeBindings() {
            var fluent = mvvmContext.OfType<FindDialogViewModel>();
            fluent.SetBinding(findTextEdit, f => f.EditValue, x => x.SearchID);
        }
    }
}
